types/events/block/index.ts

```diff
@@ -1,7 +1,11 @@
-import { BlockAddedEvent, BlockAddedMutationType } from './BlockAdded';
-import { BlockChangedEvent, BlockChangedMutationType } from './BlockChanged';
-import { BlockMovedEvent, BlockMovedMutationType } from './BlockMoved';
-import { BlockRemovedEvent, BlockRemovedMutationType } from './BlockRemoved';
+import { BlockAddedMutationType } from './BlockAdded';
+import type { BlockAddedEvent, BlockAddedMutationType } from './BlockAdded';
```

Comment:
There is a TS error:
```
Duplicate identifier 'BlockAddedMutationType'.ts(2300)
```

Reply:
Really Sorry about that. You should remove 'BlockAddedMutationType' from type import.
like this.
from - import type { BlockAddedEvent, BlockAddedMutationType } from './BlockAdded';
to - import type { BlockAddedEvent } from './BlockAdded';